### DISCUSSION TITLE
TLS only for control channel

### DISCUSSION THREAD

**Discussion by 0sorkon** [drakkan/sftpgo#1158] [Q&A] [answered]
Is it possible to apply TLS only for auth session not for data transfer in in FTP binding?
The question arose because of the old software, like Acronis 11.5, which can not auth if tls_mode 1 option is set

**Comment by drakkan** [comment, marked as answer]
Hello, this is currently not supported. However, if Acronis is unable to authenticate and you require TLS only for the control connection, nothing changes for the authentication phase. Try to enable debug logging on both side to better understand the issue.

Please read the Acronis manual here

https://dl.acronis.com/u/pdf/ABR11.5A_userguide_en-US.pdf

page 64. Only plain FTP seems supported, but you can use SFTP. You are paying for Acronis and using SFTPGo for free, please ask to their support, thanks
